kolibri/plugins/coach/assets/src/modules/exerciseDetail/handlers.js

```diff
@@ -47,18 +47,13 @@ function showExerciseDetailView({ learnerId, exerciseId, attemptId = null, inter
         interactionIndex,
         learnerId,
       });
-      return store
-        .dispatch('exerciseDetail/setAttemptLogs', {
-          learnerId,
-          exercise,
-        })
-        .then(attemptLogs => {
-          // No attemptId was passed in, so we should trigger a url redirect
-          // to the first attempt.
-          if (attemptId === null) {
-            return attemptLogs[0].id;
-          }
-        });
+      return store.dispatch('exerciseDetail/setAttemptLogs').then(attemptLogs => {
+        // No attemptId was passed in, so we should trigger a url redirect
+        // to the first attempt.
+        if (attemptId === null) {
+          return attemptLogs[0].id;
```

Comment:
we may want to explicitly check that there is at least one item here and bail if not

Reply:
Can check - any idea of what the graceful fallback should be?

Reply:
Also, with @ralphiee22's PR to set started only when exercises have attempts, this should only happen if someone manually constructs a bad URL, in which case I think it'll just show the error page. This may be sufficient?

Reply:
yeah sounds good